neuros/board/board_interface.py

```diff
@@ -0,0 +1,32 @@
+# neuros.board.board_interface.py
+
+from brainflow.board_shim import BoardShim, BrainFlowInputParams, BoardIds
+
+
+class BoardInterface:
+    """Basic interface for EEG board interaction."""
+
+    def __init__(self, board_id: int = BoardIds.SYNTHETIC_BOARD):
+        self.board_id = board_id
+        self._board = None
+
+    def start(self):
```

Comment:
When would I want to pass around an *un*started board? RAII, https://wiki.c2.com/?InitializationIsResourceAcquisition

Reply:
TIL, and this is a good point.